### PR TITLE
Added oriscus cavum glyphs.

### LINKED ISSUE #724: oriscus (& reversus) cavum
I recently got a small request from Antiphonale Monasticum project: the possibility to typeset empty oriscus and oriscus reversus, on the same model as the punctum cavum we currently have.

### PR DESCRIPTION
**Pull request by henryso**
Fixes #724.

I think the glyphs in greciliae and parmesan are passable, but the shape of the oriscus in gregorio has some irregular control points, so I had a lot of trouble making it look right.  Any fixes are welcomed.

Other that this, I think this is ready for review and merge if satisfactory.

All current tests pass, and I updated some tests to include the new figures.

### CODE REVIEW

**Comment by eroux** [comment]
Ok for me, thanks a lot!

**Comment by eroux** [comment]
In fact, I've just asked the person who asked me, is that ok to wait for her answer? Maybe the lines are too thin... I admit it's a bit strange as they have the same width as others, but they look thinner...

File: src/struct.c [hunk 1]
```diff
@@ -298,6 +297,25 @@ static void fix_punctum_cavum_inclinatum_liquescentia(gregorio_note *const note)
     }
 }
 
+static void fix_oriscus_cavum_liquescentia(gregorio_note *const note)
+{
+    note->u.note.liquescentia &= TAIL_LIQUESCENTIA_MASK;
+    switch (note->u.note.liquescentia) {
+    case L_AUCTUS_DESCENDENS:
+        note->u.note.liquescentia = L_AUCTUS_ASCENDENS;
+        // fall through
```
Comment (eroux) [inline src/struct.c]: I have a compilation error on this comment

**Comment by eroux** [comment]
I got an aswer: the lines are indeed too think (for greciliae only), here is a comparison with other cavum glyphs:

cavum

I admit I'm a bit surprised as the thickness is the same, but the eye doesn't realize that apparently... do you think you could make lines a bit thicker?

**Comment by henryso** [comment]
I'm a bit confused.  You say the lines are too thick, but then you ask that I make the lines thicker?

**Comment by henryso** [comment]
Done.  Let's not merge this until the person is happy with the shape.

**Comment by henryso** [comment]
Sample:

image

**Comment by rpspringuel** [comment]
Looks better, but the upper right-hand corner of the oriscus cavum looks a bit thin.

**Comment by henryso** [comment]
Updated.  How does this look?

image

**Comment by rpspringuel** [comment]
To my eye that looks perfect.

**Comment by henryso** [comment]
Thanks for helping me to improve the shape.  Let's see if @eroux's contact agrees.

**Comment by eroux** [comment]
Sorry, the words "thin" and "thick" are confusing for me! Anyway, this looks perfect, merging

**Pull request merged**